convert.go

```diff
@@ -94,7 +94,7 @@ func InterfaceToVariable(input interface{}) (ast.Variable, error) {
 		}
 
 		return ast.Variable{
-			Type:  ast.TypeList,
+			Type:  ast.TypeList{ast.TypeAny},
```

Comment:
This function is used heavily in Terraform for converting user input and decoded state values into `ast.Variable`.